magento/utils/client/client.ts

```diff
@@ -136,4 +139,20 @@ export interface API {
     response: [];
     body: SetShipping;
   }
+
+  "GET /rest/:reviewUrl/:productId": {
+    response: ReviewsAmastyAPI
+    searchParams: {
+      reviewUrl: string,
+      productId: string
+    };
+  };
+  
+  "POST /rest/:reviewUrl": {
+    response: SubmitReviewAmastyAPI
+    searchParams: {
+      reviewUrl: string,
+    };
+    body: SubmitReviewAmastyBody
+  }
```

Comment:
coloque a URL da doc desses endpoints

Reply:
Nao tem doc, essas sao custom

Reply:
@aka-sacci-ccr custom de onde ? se for magento tem que ter o site no minimo